deployments/anvilbox/environment.py

```diff
@@ -53,7 +53,10 @@ def mkdict(items):
 
 
 anvil_sources = mkdict([
-    ('encode', mksrc('datarepo-e8d615a1', 'AnVIL_ENCODE_default_v1', 31975))
+    ('DS_BAV_IRB_PUB_RD', 'tdr:datarepo-b0924e15:snapshot/ANVIL_CMG_UWASH_DS_BAV_IRB_PUB_RD_202209011709:/1'),
```

Comment:
We should use `mksrc()` here. Nate and I settled on a naming convention for the snapshots. To derive the key, `mksrc` should split the snapshot name on `_` and use `[1:-1]` of the resulting list.